scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -1559,6 +1560,20 @@
                (counters/dec! in-order-queue-counter)
                (body-fn))))))
 
+(defn prepare-match-trigger-chan
+  "Calculate the required interval for match-trigger-chan and start the chimes"
+  [match-trigger-chan pools]
+  (let [{:keys [per-pool-match-interval-millis global-min-match-interval-millis]} (config/offer-matching)
+        match-interval-millis (-> per-pool-match-interval-millis 
+                                  (/ (count pools)) 
+                                  int 
+                                  (max global-min-match-interval-millis))]
+    (when (= match-interval-millis global-min-match-interval-millis)
+      (log/warn "match-trigger-chan is set to the global minimum interval of " global-min-match-interval-millis " ms. "
```

Comment:
Like this!

Reply:
hmm, I thought I renamed already. renaming.